proto/protovalidate/buf/validate/validate.proto

```diff
@@ -2711,11 +2718,18 @@ message StringRules {
     //   string value = 1 [(buf.validate.field).string.hostname = true];
     // }
     // ```
-    bool hostname = 13 [(priv.field).cel = {
-      id: "string.hostname",
-      message: "value must be a valid hostname",
-      expression: "this.isHostname()",
-    }];
+    bool hostname = 13 [
+      (priv.field).cel = {
+        id: "string.hostname",
+        message: "value must be a valid hostname",
+        expression: "this.isHostname() || this == ''",
```

Comment:
nit: does ordering matter in evaluation? Would `this == '' || this.isHostname()` be more efficient?

Reply:
CEL does short circuit evaluation so technically yes.

Reply:
Fixed in [881cbd7](https://github.com/bufbuild/protovalidate/pull/133/commits/881cbd747bbe43ed05be5f61b0e5e81cf7e63d04)